OpenSim/Simulation/Manager/Manager.h

```diff
@@ -56,15 +56,15 @@ class ControllerSet;
  * using setIntegrator().
  * 
  * In order to prevent an inconsistency between the Integrator and TimeStepper,
- * we only create a TimeStepper once, specifically at the first call to
- * integrate(SimTK::State&, double). To ensure this, the Manager will throw 
- * an exception if setModel() or setIntegrator() is called after 
- * integrate(SimTK::State&, double) has been called at least once.
+ * we only create a TimeStepper once, specifically when we call
+ * initialize(SimTK::State&). To ensure this, the Manager will throw
+ * an exception if initialize(SimTK::State&) is called more than once. Note 
+ * that any subsequent changes to the SimTK::State passed into 
+ * initialize(SimTK::State&) will not affect the simulation.
```

Comment:
Suggested change in wording: "Note that any subsequent changes to the SimTK::State passed into  initialize(SimTK::State&) will not affect the simulation." -> "Note that editing the `SimTK::State` after calling `initialize(SimTK::State&)` will not affect the simulation." (I was initially confused by "subsequent").